vmdb/app/models/miq_automate_worker.rb

```diff
@@ -0,0 +1,11 @@
+class MiqAutomateWorker < MiqQueueWorkerBase
+  self.default_queue_name     = 'automate'
+  self.check_for_minimal_role = false
+  self.workers                = -> { MiqServer.minimal_env? ? 1 : worker_settings[:count] }
+  REQUIRED_ROLE = 'automate'
+  self.required_roles = [REQUIRED_ROLE]
```

Comment:
Maybe make the following change (put role info next to queue name and drop the single use constant?)
```ruby
  self.default_queue_name     = 'automate'
  self.required_roles         = 'automate'
  self.check_for_minimal_role = false
  self.workers                = -> { MiqServer.minimal_env? ? 1 : worker_settings[:count] }
```